### ISSUE TITLE
Roles

### ISSUE THREAD

**Issue by tasermonkey** [walkeriniraq/twitarr#240]
Simple user roles to ease the ability of announcements about event changes, etc.

Something like a Home Office Role (which wouldn't have all the powers of Admin) that can add announcements, edit events (when the event issue is completed), etc.

**Comment by Wingar** [comment]
RBAC is easy as pie. I'll throw something together when I get home.

**Comment by tasermonkey** [comment]
Yeah, I just want to get my list that I have written down, that you haven't added yet to the issues.
